admin/config-ui/class-configuration-endpoint.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * @package WPSEO\Admin\ConfigurationUI
+ */
+
+/**
+ * Class WPSEO_Configuration_Endpoint
+ */
+class WPSEO_Configuration_Endpoint {
+
+	const REST_NAMESPACE = 'yoast/v1';
+	const ENDPOINT_RETRIEVE = 'get-config';
```

Comment:
Two things:
1. In a REST sense, one endpoint for both get and set is usual. Get would be represented by the GET endpoint. Set would be represented by the POST endpoint. They would then be on the same URL. Which could be `config`.
2. `get-config` and `set-config` are too broad. `onboarding/get-config` and `onboarding/set-config` are more specific. We might want to introduce a general `config` endpoint in the future. Combining with the first point the endpoint would be `onboarding/config`.